cmd/wavelet/cli.go

```diff
@@ -141,6 +142,62 @@ func NewCLI(client *skademlia.Client, ledger *wavelet.Ledger, keys *skademlia.Ke
 			Aliases: []string{"quit", ":q"},
 			Action:  a(c.exit),
 		},
+		{
+			Name:   "update-parameters",
```

Comment:
Update cmd to `update-param` w/ alias `up <key> <val>`.

Reply:
ok, but it will be `update-params` since it allows to update more than one parameter at a time.
if no value provided for any parameter, default one will be used. thats because values to cli passed once on it's creation and then reused during each command enter.
its maybe not really convenient but its the simplest approach.